tools/run_tests/run_tests_matrix.py

```diff
@@ -252,7 +261,7 @@ def _create_portability_test_jobs(extra_args=[], inner_jobs=_DEFAULT_INNER_JOBS)
                               configs=['dbg'],
                               platforms=['linux'],
                               arch='default',
-                              compiler='python3.4',
+                              compiler='python3.5',
```

Comment:
With this change, `tools/run_tests/helper_scripts/build_python.sh` is executed with python3.5 as the version of Python to  use for setting up the virtualenv, but Python 3.5 isn't installed on the Dockerimage, which this script is running in. 

I'm not too familiar with `tools/run_tests/helper_scripts/build_python.sh`, but it looks like we would have to use Python 2.7 or 3.4 to set up the virtualenv that would allow Python 3.5 to be run. (I think this is how virtualenv works, but I'm not a Python guru)